docs/src/main/sphinx/release/release-469.md

```diff
@@ -0,0 +1,125 @@
+# Release 469 (8 Jan 2025)
+
+## General
+
+* Refrain from join pushdown for modified tables. ({issue}`24447`)
+* Fix parsing of negative `0x`, `0b`, `0o` long literals. ({issue}`24601`)
+
+## Security
+
+## Web UI
+
+## JDBC driver
+
+* Add `physicalInputTimeMillis` to `io.trino.jdbc.QueryStats`. ({issue}`24604`)
+* Fix loading spooled segment data when cluster is secured. ({issue}`24595`)
+
+## Server RPM
+
+## Docker image
+
+## CLI
+
+* Fix loading spooled segment data when cluster is secured. ({issue}`24595`)
+
+## BigQuery connector
+
+## Blackhole connector
+
+## Cassandra connector
+
+## ClickHouse connector
+
+## Delta Lake connector
+
+* Improve performance of reading from new Delta Lake tables by compressing files
```

Comment:
fyi .. a bit reworded @raunaqmorarka

Reply:
It applies to new files in existing tables as well since compression is not a table property